flask_annex/s3.py

```diff
@@ -84,7 +84,7 @@ def save_file(self, key, in_file):
                 in_file, self._bucket_name, key, extra_args,
             )
 
-    def send_file(self, key):
+    def send_file(self, key, **kwargs):
```

Comment:
I think what @cancan101 meant (hey man!! 👋 ) was

```suggestion
    def send_file(self, key, *, no_redirect=False):
```

see https://www.python.org/dev/peps/pep-3102/

Reply:
also `redirect=True` would probably read easier than the double negation.

Reply:
in a broader sense, this is a weird/awkward API. there's no particular benefit to adding a kwarg here, because it's not part of the upstream API.

in general, adding boolean flags that change high-level behavior is an anti-pattern in API design.

as such, it'd probably be best to factor out a `generate_presigned_url` method, and implement `send_file` as a small wrapper around that.